workflow/metrics/test_exporter.go

```diff
@@ -200,6 +217,16 @@ func (t *TestExporter) GetFloat64GaugeValue(name string, attribs *attribute.Set)
 	return gauge.Value, err
 }
 
+// GetFloat64GaugeValue returns an otel int64 gauge value
+// for reading in a test
```

Comment:
```suggestion
// GetInt64GaugeValue returns an otel int64 gauge value for test reads
```

- `Float` -> `Int`
- why not one line it?
- simplify a bit -- I also got a little confused by "reading in a test" as I interpreted that initially as "I read the test into the record"